src/Agent.Worker/WorkerCommandManager.cs

```diff
@@ -52,6 +53,14 @@ public bool TryProcessCommand(IExecutionContext context, string input)
             IWorkerCommandExtension extension;
             if (_commandExtensions.TryGetValue(command.Area, out extension))
             {
+                if (!extension.SupportedHostType.IsNullOrEmpty() &&
```

Comment:
@TingluoHuang - Agree that we should update document about commands that aren't supported in Release. But I don't know if customers will always look at the document to figure out why the command is not found when run within Release when it works in Build. I would prefer if we keep error message here more specific.